agg.pretty/ADP2389ACPZ-R7.kicad_mod

```diff
@@ -0,0 +1,186 @@
+(module ADP2389ACPZ-R7 (layer F.Cu) (tedit 5A7B854C)
+  (fp_text reference REF** (at 0.01 -3.91) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value ADP2389 (at 0 4) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start 2.627 -2.232739) (end 2.627 -2.627) (layer F.SilkS) (width 0.15))
+  (fp_line (start 2.232739 2.627) (end 2.627 2.627) (layer F.SilkS) (width 0.15))
+  (fp_line (start -2.627 2.232739) (end -2.627 2.627) (layer F.SilkS) (width 0.15))
+  (fp_line (start -2.5 -2.5) (end -2.5 2.5) (layer Dwgs.User) (width 0.1524))
```

Comment:
There should be a part drawing on F.Fab and not on Dwgs.User. The lines should all be 0.01mm and use a circle for pin 1 indication, as with all the other QFNs in agg-kicad.